docs/en/user/user-documentation.md

```diff
@@ -0,0 +1,92 @@
+
+After creating a new CKAN Registry Page, the normal page editing options are available to the content editor, along with a new tab called "Data". This tab will allow one to paste a CKAN datastore resource URL into the "Data source URL" field.
+
+This field will then load data from CKAN, providing a list of resources available on that dataset if applicable, and pre-selecting one if the URL was for a resource rather than a dataset.
+
+![Selecting a resource from a data set](resourceSelection.png)
+
+Once a valid resource has been chosen, and the page saved, SilverStripe will import a list of all the fields available on that resource, listing them down the page, and a new tab will appear for the page - "Filters".
+
+![Data tab after a resource is saved](populatedColumns.png)
+
+To change the resource after the page has been saved, click the pencil icon next to the dataset title. **Changing the resource and saving the page will reload _all_ configuration to defaults**
+
+![Changing the resource data is pulled from](changeResource.png)
+
+The list of fields by default has quick references for "Results" and "Detail", the former disabled by default and the latter enabled by default for all records. These two options will be explained in detail in the next section.
+
+    **Note** There must be at least one record with the "Results" column ticked, or no data will be displayed to visitors.
+
+Configuration:
+![Configuring fields to show in the results](inResultsEnabled.png)
+Results on the page:
+![Fields showing in the results](inResultsEnabled-frontend.png)
+
+## Fields
+
+The list of Fields will be shown to a visitor as columns on the table, and there are a number of options available to the CMS editor to configure.
+
+![Column configuration](configureColumn.png)
+
+Each field can be configured as such:
+
+* The **Original label** is not editable, and serves to indicate the source of the data as coming from the CKAN resource.
+* The **Readable label** allows the label/name to be altered to be more readable if the one imported from CKAN is undesirable for any reason
+* If **Show In Results** is checked then this field will be shown as a column on the field site visitors will see. Otherwise it will not be shown. At least one field must be selected to be shown to the user.
+* If **Show in Detail** is checked, this field will be shown in the detail view that opens when a visitor clicks a row on the table shown to them. This is enabled for all fields by default.
+* If **Only show one value if duplicates exist** removes one of two (or more) results that have the same data in the result set shown to the website visitor
+* **Presented order** allows an editor to change the ordering of columns with more precision than the drag and drop provided on the GridField. E.g. it is possible to move a column between page of a GridField
```

Comment:
```suggestion
* **Presented order** allows an editor to change the ordering of columns with more precision than dragging and dropping items in the CMS table. This also makes it easy to move items between pages.
```

Note that GridField is a developer term that CMS users are likely to be unfamiliar with